Examples/Tests/space_charge_initialization/inputs_3d

```diff
@@ -1,5 +1,5 @@
 max_step = 1
-amr.n_cell =  64 64 64
+amr.n_cell =  128 128 128
```

Comment:
The resolution had to be increased in order to mitigate the effects of filtering.
(The previous resolution was very low, with the beam being only covered with ~2-3 cells.)